tests/e2e_tests/integration/01-login.spec.ts

```diff
@@ -52,6 +47,14 @@ test.describe('Login', () => {
       expect(loginVisible).toBeTruthy();
     });
 
+    test('Logs out using UI', async ({ loggedInPage: page }) => {
+      await page.waitForSelector(loggedInText);
```

Comment:
this will work because this test will wait for a successful login from the other test case?

Reply:
if you have code to remove, do not wait for my permission :)